lib/Doctrine/DBAL/Driver/Mysqli/MysqliConnection.php

```diff
@@ -97,6 +97,12 @@ public function getWrappedResourceHandle()
      */
     public function getServerVersion()
     {
+        // Hack for mariadb
+        $serverInfos = $this->_conn->get_server_info();
+        if (false !== stripos($serverInfos, 'mariadb')) {
```

Comment:
Spacing needed

Reply:
Is a `strtolower()` needed here? I would suggest so

Reply:
Mmm, `strtolower` not needed. Using `stripos`. Am I wrong somewhere ?